src/containers/pages/FocusInvestigation/active/index.tsx

```diff
@@ -338,18 +340,24 @@ interface DispatchedStateProps {
 }
 
 /** map state to props */
-const mapStateToProps = (state: Partial<Store>): DispatchedStateProps => {
-  const caseTriggeredPlans = getPlansArray(
+const mapStateToProps = (state: Partial<Store>, ownProps: any): DispatchedStateProps => {
+  const caseTriggeredPlans = getFilteredPlansArray(
     state,
     InterventionType.FI,
-    [PlanStatus.ACTIVE, PlanStatus.COMPLETE],
-    CASE_TRIGGERED
+    [PlanStatus.ACTIVE, PlanStatus.DRAFT],
```

Comment:
The plans here should be filtered based on `PlanStatus.ACTIVE, PlanStatus.COMPLETE`. i believe that was the change that was in master before you rebased.